sheetstorm/tests/test_unit_generate_test_data.py

```diff
@@ -0,0 +1,207 @@
+
+        self.number_of_suspended_projects_to_create = (
+            self.options.get(ProjectType.SUSPENDED.name) - Project.objects.filter_suspended().count()
+        )
+        self.number_of_active_projects_to_create = (
+            self.options.get(ProjectType.ACTIVE.name) - Project.objects.filter_active().count()
+        )
+        self.number_of_completed_projects_to_create = (
+            self.options.get(ProjectType.COMPLETED.name) - Project.objects.filter_completed().count()
+        )
+
+    def tearDown(self):
+        Project.objects.all().delete()
+
+    @parameterized.expand(
+        [(ProjectType.SUSPENDED.name, 2), (ProjectType.ACTIVE.name, 3), (ProjectType.COMPLETED.name, 5)]
+    )
+    def test_number_of_projects_to_create_should_be_a_difference_of_number_of_projects_in_database_and_definite_number(
+        self, project_type, expected_result
+    ):
+        self.assertEqual(Command._compute_number_of_projects_to_create(self.options, project_type), expected_result)
+
+    def test_compute_number_of_projects_to_create_function_should_raise_error_if_project_type_does_not_exist(self):
+        with self.assertRaises(UnsupportedProjectTypeException):
+            Command._compute_number_of_projects_to_create(self.options, "error")
+
+    @parameterized.expand([ProjectType.SUSPENDED.name, ProjectType.ACTIVE.name, ProjectType.COMPLETED.name])
+    def test_compute_number_of_projects_to_create_function_should_return_0_if_number_of_projects_is_not_specified(
+        self, project_type
+    ):
+        self.assertEqual(Command._compute_number_of_projects_to_create({project_type: None}, project_type), 0)
+
+    def test_number_of_projects_to_create_should_be_set_and_equal_to_computed_number(self):
+        expected_options = {
+            ProjectType.ACTIVE.name: self.number_of_active_projects_to_create,
+            ProjectType.SUSPENDED.name: self.number_of_suspended_projects_to_create,
+            ProjectType.COMPLETED.name: self.number_of_completed_projects_to_create,
+        }
+        self.assertDictEqual(Command(self.options)._set_number_of_projects_to_create(self.options), expected_options)
+
+    def test_start_date_should_be_of_datetime_type_and_set_in_past(self):
+        start_date_returned_by_function = Command._create_start_date()
+        self.assertTrue(isinstance(start_date_returned_by_function, timezone.datetime))
+        self.assertLess(start_date_returned_by_function, timezone.now())
+
+    def test_stop_date_should_be_of_datetime_type_and_set_in_past(self):
+        stop_date_returned_by_function = Command._create_stop_date()
+        self.assertTrue(isinstance(stop_date_returned_by_function, timezone.datetime))
+        self.assertLess(stop_date_returned_by_function, timezone.now())
+
+    def test_suspended_projects_should_have_suspended_property(self):
+        self.assertTrue(Command()._set_project_factory_parameters(ProjectType.SUSPENDED.name).get("suspended"))
```

Comment:
You are using `self.assertTrue` and `self.assertFalse` too often. You should be more precise during testing and check exactly what you want to check.

Reply:
Done. I left `self.assertTrue` or `self.assertFalse` only for case when function return bool.